borbskeeper/AppDelegate.m

```diff
@@ -26,6 +27,11 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     }];
     
     [Parse initializeWithConfiguration:config];
+ 
+    UNUserNotificationCenter* center = [UNUserNotificationCenter currentNotificationCenter];
+    [center requestAuthorizationWithOptions: (UNAuthorizationOptionAlert + UNAuthorizationOptionSound) completionHandler:^(BOOL granted, NSError * _Nullable error) {
+        
```

Comment:
No; the phone will not send push notifications if permissions are not granted. 

I stored and deleted notifications as usual even when permissions aren't granted so that if the user turns them on later, notifications for previously created tasks will still occur.